pkg/client/cache/listers.go

```diff
@@ -87,6 +89,20 @@ func (s storePodsNamespacer) List(selector labels.Selector) (pods api.PodList, e
 	return list, nil
 }
 
+// A verioned duplication.
+func (s storePodsNamespacer) V1List(selector labels.Selector) (pods v1.PodList, err error) {
```

Comment:
This is an temporary arrangement. All functions in this file will target versioned objects once the migration is done.